notebooks/en/rag_with_knowledge_graphs_neo4j.ipynb

```diff
@@ -0,0 +1,644 @@
+{
```

Comment:
<p>can you explain what a Cypher statement is with a small sentence, you mention it on top as well, might be good for beginners </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
added a short explanation "[...] generates Cypher statements (queries for graph databases like Neo4j) from user input [...]"